app/models/team.rb

```diff
@@ -6,9 +6,13 @@ class Team < ActiveRecord::Base
   validates :name, presence: true
 
   def self.make(team, summoner)
-    team = Team.find_or_create_by(full_Id: team["fullId"],
+    roster = team["roster"]["memberList"]
+    new_team = Team.find_or_create_by(full_Id: team["fullId"],
                                   name: team["name"],
                                   tag: team["tag"])
-    Roster.make_connection(team, summoner)
+    roster.each do |player|
+      summoner = Summoner.add(player['playerId'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.